server.js

```diff
@@ -25,8 +25,14 @@ app.use(express.static('public'));
 app.engine('handlebars', exphbs({
   defaultLayout: 'main',
   helpers: {
+    upperCase: function(str) {
+      return str.charAt(0).toUpperCase() + str.slice(1);
+    },
     momentFromNowTime: function (time) {
       return moment(time).fromNow();
+    },
+    formatDateForEvents: function (time) {
```

Comment:
@catchbb14  nice addition. css would not be able to do this, and we do this to convert with moment.js on the server side.